client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -176,6 +180,32 @@ const PreviewSiteModalItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 	);
 };
 
+const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
+	const { __ } = useI18n();
+	const hasAtomicFeature = useSafeSiteHasFeature( site.ID, WPCOM_FEATURES_ATOMIC );
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const plan = site.plan;
+	const isSiteOwner = site.site_owner === userId;
+
+	if ( ! hasAtomicFeature || ! isSiteOwner || ! plan ) {
+		return null;
+	}
+
+	const copySiteHref = addQueryArgs( `/setup/copy-site`, {
+		productSlug: plan.product_slug,
+		sourceSlug: site.slug,
+		sourceUrl: site.URL,
+	} );
+	return (
+		<MenuItemLink
+			href={ copySiteHref }
+			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_copy_site' ) }
+		>
+			{ __( 'Create a copy of this site' ) }
```

Comment:
```suggestion
			{ __( 'Copy site' ) }
```

Can we shorten this?

Reply:
Done here https://github.com/Automattic/wp-calypso/commit/330616717a8cf5267100d9a8778417b89153f3bd
Don't you think that `Copy site` is a vague description? Users may be confused and think they will copy the URL?
I suggest using `Duplicate site`.

Reply:
> Don't you think that `Copy site` is a vague description?
> Users may be confused and think they will copy the site URL to the clipboard to paste it somewhere.

That's a good point. We can iterate on it.